ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityValidatorBuilder.service.ts

```diff
@@ -62,6 +64,9 @@ export class ActivityValidatorBuilder {
         if (questionBlock.questionType === QuestionType.Date) {
             rules.push(new ActivityDateNavyValidationRule(questionBlock, this.dateService));
         }
+        if (questionBlock instanceof ActivityTextQuestionBlock && questionBlock.confirmEntry) {
```

Comment:
Text questions with the `EMAIL` input type can also have `confirmEntry`, but it looks like `ActivityMatchValidationRule` is only meant for `TEXT` input type? I think this is causing some unwanted error messages. For example, the Brain study has a `EMAIL` question with confirmation in the Child Contact activity and I see this:

![Screen Shot 2021-04-15 at 11 45 49 AM](https://user-images.githubusercontent.com/5713833/114899308-d9f6d700-9de0-11eb-956c-a88a7ba15a28.png)

Reply:
Thanks for that! I'll only apply this validation rule only for `Text` input type.

Reply:
Done. It shouldn't be a problem for questions with `Email` input type from now on.